application/src/main/kotlin/net/corda/v5/application/uniqueness/model/UniquenessCheckStateRef.kt

```diff
@@ -9,7 +9,9 @@ import net.corda.v5.crypto.SecureHash
  * model and is agnostic to both the message bus API and any DB schema that may be used to persist data
  * by the backing store.
  *
- * T0DO CORE-6629: This class can be removed once the UTXO ledger model gets merged
+ * Please note that this representation of a state ref is entirely different from the Ledger specific `StateRef` class.
+ * This class represents a state ref that the uniqueness checker will process and might be different from the UTXO data
+ * model.
```

Comment:
```suggestion
 * Please note that this representation of a state ref is entirely different from the Ledger specific
 * [StateRef][net.corda.v5.ledger.utxo.StateRef] class. This class represents a state ref that the
 * uniqueness checker will process and may differ from the UTXO ledger data model.
```
Made it a bit more succinct, linked to the ledger class, and matched line lengths with the existing earlier comments,